<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Companies</class>
 <widget class="QDialog" name="Companies">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Datos</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>RUN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_rut"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Raz√≥n Social</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_name"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>491</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_newCompany">
      <property name="text">
       <string>Nuevo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_update">
      <property name="text">
       <string>Actualizar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBox_search">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_search">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_new">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
